apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -146,26 +147,26 @@ const initSelectedConversationsEpic: AppEpic = (action$, state$) =>
       const optionsReceived = OverlaySelectors.selectOptionsReceived(
         state$.value,
       );
+      const previousRoute = UISelectors.selectPreviousRoute(state$.value);
 
-      return concat(
-        iif(
-          () => !isOverlay,
-          of(ConversationsActions.getSelectedConversations()),
-          EMPTY,
-        ),
-        iif(
-          () => isOverlay && !!optionsReceived,
-          of(ConversationsActions.getSelectedConversations()),
-          EMPTY,
+      return iif(
+        () => !isOverlay || !!optionsReceived,
+        of(
+          ConversationsActions.getSelectedConversations({
```

Comment:
Would it be better to call the createNewConversations action instead of the new flag and conditions? Or is there some problem with this?